lnst/Recipes/ENRT/VlansOverBondRecipe.py

```diff
@@ -309,3 +310,7 @@ def pause_frames_dev_list(self):
         """
         host1, host2 = self.matched.host1, self.matched.host2
         return [host1.eth0, host1.eth1, host2.eth0]
+
+    @property
+    def vf_trust_device_list(self) -> list[RemoteDevice]:
+        return [sriov_devices.phys_dev for sriov_devices in self.vf_config[self.matched.host1]]
```

Comment:
same problem with list of devices here.

Reply:
Updated per https://github.com/LNST-project/lnst/pull/381#discussion_r1805965954